RDKOSS-218: Parse and update SRCREV with overrides

maniselva006c opened the pull request:
Reason for change:  Parse SRCREV parameters configured with overrides and update them sha valaue if tag is used.

Commit:
RDKOSS-218: Parse and update SRCREV with overrides

Reason for change:  Parse SRCREV parameters configured with overrides and
update them sha valaue if tag is used.

Signed-off-by: mselva006c <mani_selvaraj@comcast.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/meta-rdk-auxiliary?pullRequest=18) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/meta-rdk-auxiliary?pullRequest=18) before we can accept your contribution.<br/><hr/>**mselva006c** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/meta-rdk-auxiliary?pullRequest=18) it.</sub>

Commit:
RDKOSS-218: Fix incomplete tag to SHA conversion in linux-meson

Reason for the change: The "linux-meson" recipe fails to trigger the "bb.event.RecipeParsed" event,
causing an incomplete conversion of the tag to SHA in SRCREV. As a result, the build fails.
To resolve this issue, the tag to SHA conversion implementation has been moved to a Python anonymous API.
This API is now automatically called at the end of the recipe parsing process, ensuring a complete conversion.

Signed-off-by: mselva006c <mani_selvaraj@comcast.com>

maniselva006c commented:
Triggered below verification builds 
meta-rdk-auxiliary 81dc2cd30885131c5b2c83a3b5f4fa8222f89a45
https://rdkjenkins-e.stb.r53.xcal.tv/jenkins/job/RTK-XIONE-Vendor-Build/1953/
https://rdkjenkins-e.stb.r53.xcal.tv/jenkins/job/ELEMENT-TEONE-Vendor-Build/1693/
https://rdkjenkins-e.stb.r53.xcal.tv/jenkins/job/ELEMENT-TEONE-Middleware-Build/16730/
https://rdkjenkins-e.stb.r53.xcal.tv/jenkins/job/RTK-XIONE-Middleware-Build/15182/
